src/main/java/core/basesyntax/strategy/ReturnOperation.java

```
@@ -0,0 +1,12 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.common.FruitTransaction;
+import core.basesyntax.db.Storage;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void apply(FruitTransaction fruitTransaction) {
+        Storage.storage.merge(fruitTransaction.getFruit(),
+                fruitTransaction.getQuantity(), Integer::sum);
```

No validation of `fruit` (non-null/non-empty) or `quantity` (non-negative) before updating Storage. The project guidance requires anticipating invalid input (e.g., quantity < 0) and throwing informative exceptions: "... Incorrect data in the input file, for example, quantity less than zero ..."  and "Don't ignore exceptions in try-catch blocks ... Let's rethrow a RuntimeException with an informative message and exception object." . Before calling `Storage.storage.merge` / `put`, validate `fruitTransaction.getFruit()` and `fruitTransaction.getQuantity()` and throw descriptive RuntimeExceptions when invalid.